testdata/02-unittest.tdir/02-unittest.test

```diff
@@ -16,11 +16,16 @@ else
 fi
 VALGRIND_FLAGS="--leak-check=full --show-leak-kinds=all"
 
+## START -- Loop over unit tests
+##
+for unit_cmd in "unittest" "unbound-dnstap-socket -c"; do
```

Comment:
This builds unbound-dnstap-socket also when dnstap is not enabled, that seems to be harmless. But it also runs the '-c' test on it, when there is no enable-debug, this is going to fail non-debug compile tests.

Reply:
Great catch! I believe I solved it with the latest commit.

Reply:
Yes, that looks fixed.